internal/util/rocksmq/server/rocksmq/rocksmq_impl.go

```diff
@@ -159,6 +161,7 @@ func NewRocksMQ(name string, idAllocator allocator.GIDAllocator) (*rocksmq, erro
 		storeMu:     &sync.Mutex{},
 		consumers:   sync.Map{},
 		ackedMu:     sync.Map{},
+		readers:     sync.Map{},
```

Comment:
Reader is a map from `Topic` to `Reader`. There may be a case that multiple reader on same topic. Then the logic here is messed up.